csharp/ql/src/XMLAlertSuppression.ql

```diff
@@ -0,0 +1,74 @@
+/**
+ * @name XML alert suppression
+ * @description Generates information about alert suppressions in XML files.
+ * @kind alert-suppression
+ * @id cs/xml-alert-suppression
+ */
+
+import semmle.code.csharp.XML
+
+/**
+ * An alert suppression comment.
+ */
+class SuppressionComment extends XMLComment {
```

Comment:
It feels like this class and `SuppressionScope` should be in `XML.qll`, so it can be reused by other languages.